### PR TITLE
534 Add customer name and abbreviation to subscriptionslist

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/mapGrapghQlSubscriptionsResultToSubscriptionListItems.ts [hunk 1]
```diff
@@ -24,10 +26,13 @@ export function mapGrapghQlSubscriptionsResultToSubscriptionListItems(
...
             subscriptionId,
             note,
+            customer,
         } = subscription;
 
         const { name: productName, tag } = product;
-
+        const { fullname: customerFullname, shortcode: customerShortcode } =
+            customer;
+        customer;
```
Comment (tjeerddie) [inline packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/mapGrapghQlSubscriptionsResultToSubscriptionListItems.ts:35]: remove useless line
```suggestion
```

File: packages/orchestrator-ui-components/src/messages/nl-NL.json [hunk 2]
```diff
@@ -224,7 +224,9 @@
             "tag": "Tag",
             "startDate": "Startdatum",
             "endDate": "Einddatum",
-            "note": "Notitie"
+            "note": "Notitie",
+            "customerFullName": "Klant",
+            "customerAbbreviation": "Klantafkorting"
```
Comment (tjeerddie) [inline packages/orchestrator-ui-components/src/messages/nl-NL.json:229]:
```suggestion
            "customerAbbreviation": "Klant Afkorting"
```

**Review by tjeerddie — APPROVED** [review]

File: packages/orchestrator-ui-components/src/messages/nl-NL.json (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/messages/nl-NL.json:229]: done

File: packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/mapGrapghQlSubscriptionsResultToSubscriptionListItems.ts (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/mapGrapghQlSubscriptionsResultToSubscriptionListItems.ts:35]: Done

File: packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/WfoSubscriptionsList.tsx [hunk 3]
```diff
@@ -118,6 +118,16 @@ export const WfoSubscriptionsList: FC<WfoSubscriptionsListProps> = ({
             field: 'note',
             name: t('note'),
         },
+        customerFullname: {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/WfoSubscriptionsList.tsx:121]: Do they have to be at the most right side of the table -- position in this config object determines the location of the column

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/WfoSubscriptionsList.tsx (hunk 3 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/WfoSubscriptionsList.tsx:121]: Something for the acceptance feedback

**Pull request merged**